packages/react-dom/src/__tests__/ReactDOMEventListener-test.js

```diff
@@ -25,62 +25,67 @@ describe('ReactDOMEventListener', () => {
     ReactDOM = require('react-dom');
     // TODO: can we express this test with only public API?
     ReactDOMComponentTree = require('ReactDOMComponentTree');
-    ReactDOMEventListener = require('ReactDOMEventListener');
     ReactTestUtils = require('react-dom/test-utils');
-
-    handleTopLevel = jest.fn();
-    ReactDOMEventListener._handleTopLevel = handleTopLevel;
   });
 
   it('should dispatch events from outside React tree', () => {
-    var otherNode = document.createElement('h1');
-    var component = ReactDOM.render(<div />, document.createElement('div'));
-    expect(handleTopLevel.mock.calls.length).toBe(0);
-    ReactDOMEventListener.dispatchEvent('topMouseOut', {
-      type: 'mouseout',
-      fromElement: otherNode,
-      target: otherNode,
-      srcElement: otherNode,
-      toElement: ReactDOM.findDOMNode(component),
-      relatedTarget: ReactDOM.findDOMNode(component),
-      view: window,
-      path: [otherNode, otherNode],
-    });
-    expect(handleTopLevel.mock.calls.length).toBe(1);
+    var mock = jest.fn();
+
+    var container = document.createElement('div');
+    var stub = ReactDOM.render(<div onMouseOver={mock} />, container);
+    var node = ReactDOM.findDOMNode(stub);
+    var otherComponent = document.createElement('h1');
```

Comment:
Let's not call nodes "components". I understand this naming already existed but let's keep it clear in new code. We call DOM nodes "nodes" in newer code (or "containers" when we render into them). So `otherNode` was a good naming.